src/Services/SubscriptionService.php

```diff
@@ -143,4 +143,36 @@ public function issueInterimInvoice($subscriptionId, $data)
             ['subscriptionId' => $subscriptionId]
         );
     }
+
+    /**
+     * @param string $subscriptionId
+     * @param array|ArrayObject $params
+     *
+     * @return Invoice[]|Collection
+     */
+    public function searchUpcomingInvoices($subscriptionId, $params = [])
```

Comment:
```suggestion
    public function getUpcomingInvoices($subscriptionId, $params = [])
```
There is no search, actually...